### ISSUE TITLE
Add jumper/switch to select between I2C and SPI display

### ISSUE THREAD

**Issue by joelspadin** [zmkfirmware/zmk-uno#2]
Using solder jumpers makes it difficult to switch whether the display pins are connected to I2C or SPI. There isn't a lot of free space left, but it would be nice to have jumper headers or similar so we don't have to break out the soldering iron in order to test different types of displays.

**Comment by petejohanson** [comment]
Yeah.. let me think about how best to do this... We'd probably need to add another switching IC like we use for the direct vs matrix wire setup, but for the data lines.

**Comment by joelspadin** [comment]
Some things that could work:
- A block of 6 pins with display data in the center, SCL/SDA on one side, and SCK/MOSI on the other, so you can pick which are connected with a pair of jumpers.
    ```
    I2C        SPI
    [o o] o    o [o o]
    [o o] o    o [o o]
    ```
- A DPDT switch to do the same.
- An analog switch IC like you mentioned.
